### PR TITLE
Added width to fix IE image distorted issue #1052

### PR DESCRIPTION
**Pull request by flora8984461**
Part of #1052 

For Compatibility Page, after the fix:

image

image

Might still be a little bit not perfect on IE, but at least not distorted UI. 🤔 (I think we don't want to spend a lot on IE)

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

**Review by autumnfound — APPROVED** [review]

**Comment by flora8984461** [comment]
@autumnfound Added one more fix, re-request you > <

File: js/contributor-cards/index.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ import contributorList from '../../data/contributors/jakarta-ee-9.json';
 
 const cardList = document.querySelector('.card-list.contributor-list');
 
+if (window.NodeList && !NodeList.prototype.forEach) {
+  NodeList.prototype.forEach = Array.prototype.forEach;
+}
+
```
Comment (flora8984461) [inline js/contributor-cards/index.js]: @autumnfound Is this snippet also not allowed? this is from https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach 
If so, I will remove it and make this PR fix IE images first. And find a way to add polyfill later.

Comment (autumnfound) [reply js/contributor-cards/index.js]: Yeah we should remove that snippet. Once that is done we can look at merging this

Comment (flora8984461) [reply js/contributor-cards/index.js]: removed

**Comment by chrisguindon** [comment]
@flora8984461 Can you re-roll this PR. There is now a conflict now that I merged #1061

**Review by chrisguindon — APPROVED** [review]

**Comment by flora8984461** [comment]
conflict fixed @chrisguindon

**Pull request merged**